spec/zoneddatetime.html

```diff
@@ -1262,14 +1309,24 @@ <h1>
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-temporal-differencezoneddatetime" aoid="DifferenceZonedDateTime">
-      <h1>DifferenceZonedDateTime ( _ns1_, _ns2_, _timeZone_, _calendar_, _largestUnit_ [ , _options_ ] )</h1>
-      <p>
-        The abstract operation DifferenceZonedDateTime computes the difference between two exact times expressed in nanoseconds since the Unix epoch, and balances the result so that there is no non-zero unit larger than _largestUnit_ in the result, taking calendar reckoning and time zone offset changes into account.
-      </p>
+    <emu-clause id="sec-temporal-differencezoneddatetime" type="abstract operation">
+      <h1>
+        DifferenceZonedDateTime (
+          _ns1_: a BigInt value,
+          _ns2_: a BigInt value,
+          _timeZone_: an Object,
+          _calendar_: an Object,
+          _largestUnit_: a String,
+          optional _options_: an Object,
+        )
```

Comment:
what type does it return?

Reply:
It returns a `Duration Record`. Would it be fine to just do `: Duration Record` or is there another way of doing this?

Reply:
`: a Duration Record` seems great!

Reply:
Done :+1: